src/main/kotlin/com/hibob/academy/dao/models.kt

```diff
@@ -1,13 +1,17 @@
 package com.hibob.academy.dao
 
-import java.util.*
+import java.sql.Date
 
 data class Example(val id: Long, val companyId: Long, val data: String)
 
 data class OwnerData(val id: Long, val name: String, val companyId: Long, val employeeId: String)
 
+data class Owner(val name: String, val companyId: Long, val employeeId: String)
+
 data class PetData(val id: Long, var name: String, val type: PetType, val companyId: Long, val dateOfArrival: Date, val ownerId: Long?)
 
+data class Pet(var name: String, val type: PetType, val companyId: Long, val dateOfArrival: Date, val ownerId: Long?)
```

Comment:
I understand know, please rename this data class to something we can understand like PetCreationRequest

Reply:
Why var name and not val?

Reply:
same as in owner, i changed it